apps/dashboard/src/components/conditions-editor/variable-select.tsx

```diff
@@ -1,78 +1,10 @@
 import React, { useMemo, useState, useRef } from 'react';
-import { CheckIcon } from '@radix-ui/react-icons';
 
-import { Code2 } from '@/components/icons/code-2';
 import { Popover, PopoverAnchor, PopoverContent } from '@/components/primitives/popover';
 import { InputPure, InputRoot, InputWrapper } from '@/components/primitives/input';
 import { AUTOCOMPLETE_PASSWORD_MANAGERS_OFF } from '@/utils/constants';
-import { cn } from '@/utils/ui';
-
-const KeyboardItem = ({ children, className }: { children: React.ReactNode; className?: string }) => {
-  return (
-    <span
-      className={cn(
-        'text-foreground-400 shadow-xs text-paragraph-2xs flex h-5 w-5 items-center justify-center rounded-[6px] border border-neutral-200 px-2 py-1 font-light',
-        className
-      )}
-    >
-      {children}
-    </span>
-  );
-};
-
-type VariablesListProps = {
-  options: Array<{ label: string; value: string }>;
-  onSelect: (value: string) => void;
-  selectedValue?: string;
-  title: string;
-  hoveredOptionIndex: number;
-};
-
-const VariablesList = React.forwardRef<HTMLUListElement, VariablesListProps>(
```

Comment:
moved to a reusable component, which is used later as the Email editor suggestions dropdown for variables